src/main/scala/io/viash/functionality/dependencies/Repository.scala

```diff
@@ -60,7 +60,7 @@ object Repository {
 
     // TODO this match is totally not up to snuff
```

Comment:
I can't add a comment to the line `val repoRegex = raw"(\w+)://([A-Za-z0-9/_\-\.]+)@([A-Za-z0-9]+)".r` so I'll add it to the line below. The `repoRegex` regex is not permissive enough, because I can't match something like `github://openpipelines-bio/openpipeline@0.1.0`. Also, I expect to be able to omit the tag like `github://openpipelines-bio/openpipeline`, in line with the argument in `GitRepository` being `tag: Option[String]`.